packages/flutter/test/widgets/navigator_test.dart

```diff
@@ -5189,7 +5189,7 @@ void main() {
                             child: _LinksPage(
                               title: 'Can pop page',
                               canPop: true,
-                              onPopInvoked: (bool didPop) {
+                              onPopInvoked: (bool didPop, void result) {
```

Comment:
```suggestion
                              onPopInvoked: (bool didPop, _) {
```

Reply:
As Justin said, maybe we can get away with `Object? result` for the `void result` variables? It's a bit less weird and should work too?